multi/imxrt-multi/uart.c

```diff
@@ -232,7 +232,7 @@ static int uart_handleIntr(unsigned int n, void *arg)
 	uint32_t status = *(uart->base + statr);
 
 	/* Disable interrupts, enabled in uart_intrThread */
```

Comment:
IMHO the comment needs to be updated, you're not disabling all of the interrupts right now. Please add/update comments so the code would be understandable without checking the reference manual